tribuo-tutorials/README.md

```diff
@@ -0,0 +1,35 @@
+# Tribuo tutorials
+
+This project shows how to run [Tribuo Machine Learning](https://tribuo.org) algorithms with Conclave.
+
+The code is based on [Tribuo's tutorials](https://tribuo.org/learn/4.0/tutorials/), which cover the following examples:
+* [Classification](https://tribuo.org/learn/4.0/tutorials/irises-tribuo-v4.html)
+* [Clustering](https://tribuo.org/learn/4.0/tutorials/clustering-tribuo-v4.html)
+* [Regression](https://tribuo.org/learn/4.0/tutorials/regression-tribuo-v4.html)
+* [Anomaly Detection](https://tribuo.org/learn/4.0/tutorials/anomaly-tribuo-v4.html)
+* [Configuration](https://tribuo.org/learn/4.0/tutorials/configuration-tribuo-v4.html)
+
+In this sample the client is using Mail to communicate with the enclave.
+Each mail contains a serialized request for the enclave to execute some action, i.e.,
```

Comment:
Maybe expand on the confidential computing point we're trying to make?  eg "The scenarios that these tutorials explore is one where you want to train or utilise a model on a computer operated by a third party and where it is imperative that the third party cannot access the training data or the resulting models and that they cannot observe or influence execution. The tutorials demonstrate how to use Conclave to create secure enclaves that can operate in this manner, and how to access them remotely in a way that can verify that the models are indeed being executed in a mode that is protected from the owner of the computer on which they run".  Something like that?

Reply:
I couldn't say it better myself :eyes: I've update this section.